src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -288,7 +288,7 @@ class SessionService(
   ): Boolean {
     if (offenderNonAssociationList.isNotEmpty()) {
       val nonAssociationPrisonerIds =
-        getNonAssociationPrisonerIds(session.startTimestamp.toLocalDate(), offenderNonAssociationList)
```

Comment:
non active NA are removed by the API

Reply:
What about future active NA, e.g. active NA from  Tuesday 10th next week and expires in a month, not even sure if they work that way. The session is on Tuesday 10th (session.startTimestamp) will the new api come back with that NA?

Reply:
Nope, new service has no concept of future expiry and NOMIS has also removed it